webrtc.html

```diff
@@ -9605,6 +9606,14 @@ <h3 id="sec.identity-verify-assertion">Verifying Identity Assertions</h3>
           communicate with peers that offer a certificate that doesn't match an
           <code>a=fingerprint</code> line in the negotiated session
           description.</p>
+          <p class="note">The user agent decodes <a data-for=
+          "RTCIdentityValidationResult"></code>contents</code></a> using
+          the format described in [[!RTCWEB-SECURITY-ARCH]] and do not perform
+          string-based comparison with the original <a
```

Comment:
I don't understand why you need to say "do not perform string-based comparison with the original".  Isn't it sufficient to say that the contents are opaque and should be treated as such?  We're just asking the IdP not to touch this nicely, which should be sufficient.

That the IdP could tamper with this because we have no inbuilt mechanism to detect that isn't that relevant to a specification.  The consequence of tampering is that it later breaks something if the browser-browser protocol changes.